node_brain:
  ros__parameters:
    autostart_listening: false
    activate_movement_waiting: false
    velocity_factor_linear: 0.01
    velocity_factor_rotation: 0.01
    body_factor_height: 0.04
    min_body_height: -0.04
    max_body_height: 0.10
    joystick_deadzone: 0.04
    supply_voltage: 11.1
    supply_voltage_low: 10.5
    supply_voltage_critical_low: 10.0
    servo_voltage: 11.1
    servo_voltage_low: 10.5
    servo_voltage_critical_low: 10.0
    servo_temperature_high: 70.0
    servo_temperature_critical_high: 75.0
